src/clojure/nrepl/server.clj

```diff
@@ -203,13 +204,18 @@
        would provide this greeting upon connecting to the server, but telnet et
        al. isn't that. See `nrepl.transport/tty-greeting` for an example of such
        a function.
+   * :port-file — if specified, the path to a file where the server port will be
+       written. The file will be deleted on JVM exit. Defaults to nil (no file
```

Comment:
Hmm, shouldn't the default be `.nrepl-port`? Otherwise we'd be changing the default behavior unless I'm missing something.